deploy/eck-operator/values.yaml

```diff
@@ -223,7 +223,7 @@ config:
   # Interval between observations of Elasticsearch health, non-positive values disable asynchronous observation.
   elasticsearchObservationInterval: 10s
 
-  # ubiOnly specifies whether the operator will use only UBI container images to deploy Elastic Stack applications. UBI images are only available from 7.10.0 onward.
+  # ubiOnly specifies whether the operator will use only UBI container images to deploy Elastic Stack applications as well as its own deployment. UBI images are only available from 7.10.0 onward.
```

Comment:
Also update: https://github.com/elastic/cloud-on-k8s/blob/c1182524a6874f04f27df7b12496f889173d2d65/cmd/manager/main.go#L308 and regenerate the api-docs to synchronize `./docs/operating-eck/operator-config.asciidoc`?

Reply:
@thbkrkr I don't think these are auto-generated. I see previous manual updates https://github.com/elastic/cloud-on-k8s/pull/6391/files#diff-be6fe99c5b6e0b3cfcd8302de12ec7855fc827001d2496a7257095af8bad66d5R42. I just manually updated this.

Reply:
Sorry, thinking a bit more, it doesn't really make sense as you don't use the operator to deploy the operator. It only makes sense in the context of the Helm chart.

Also, rereading it, I wonder if "deployment" is not confusing because the operator is deployed by default in a Statefulset. IIRC a deployment is used only when deploying on OpenShift using OperatorHub.

Reply:
> Also, rereading it, I wonder if "deployment" is not confusing because the operator is deployed by default in a Statefulset.

Good point, I've re-worded it. I also removed the comment from the flag, as I agree, it didn't make sense in that context.

Reply:
I think I still see the comment in the flag (file `cmd/manager/main.go`): https://github.com/elastic/cloud-on-k8s/pull/7486/files#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633.

Reply:
> I think I still see the comment in the flag (file `cmd/manager/main.go`): https://github.com/elastic/cloud-on-k8s/pull/7486/files#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633.

My apologies, this has been removed.